che-plugin-viewer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+
+RUN printf "[nginx]\nname=nginx repo\nbaseurl=http://nginx.org/packages/centos/7/x86_64/\ngpgcheck=0\nenabled=1" > /etc/yum.repos.d/nginx.repo && \
```

Comment:
Are you the author of this file or it is copied from somewhere?

Reply:
I resued this repo  - https://github.com/jeff-phillips-18/catalog-test-app

Reply:
It's licensed with Apache. Seems OK. @skabashnyuk @l0rd WDYT?

Reply:
We can licensed it under EPL 2.0. No issues.